spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -141,4 +141,48 @@ class PowerShellManager; end
     end
   end
 
+  describe "when output is written to a PowerShell Stream" do
```

Comment:
It looks like PowerShell specially parses requests to merge streams (and must have special internal handling), and doesn't allow arbitrary stream / handle merging as I've suggested (based on what `cmd.exe` allows).  I think as long as something like `3>&1` works OK, then we're probably fine.